application/models/organization.go

```diff
@@ -55,10 +62,19 @@ func (o *Organization) ValidateUpdate(tx *pop.Connection) (*validate.Errors, err
 }
 
 func (o *Organization) GetAuthProvider() (auth.Provider, error) {
+
 	if o.AuthType == AuthTypeSaml {
 		return saml.New([]byte(o.AuthConfig))
 	}
 
+	if o.AuthType == AuthTypeGoogle {
+		gCfg := google.GoogleConfig{}
+		json.Unmarshal([]byte(o.AuthConfig), &gCfg)
```

Comment:
The SAML provider does the Unmarshal inside New. For consistency, should they be the same?

Reply:
Good point.